default/stringtables/en.txt

```diff
@@ -7917,6 +7920,11 @@ invalid ship part class
 PC_DIRECT_WEAPON
 Direct Weapon
 
+PC_DIRECT_WEAPON_DESC
+'''Direct-fire weapons acquire targets without third-party assistance (a friendly ship or planet).
```

Comment:
I assumed `Direct Weapon` referred to the opposite of Indirect-fire, so this was an attempt at a definition without mentioning Line-of-sight.

How about `A Direct Weapon discharges energy or a projectile directly towards a target.` ?